include/deal.II/grid/grid_generator.h

```diff
@@ -1746,6 +1763,102 @@ namespace GridGenerator
     Triangulation<dim, spacedim> &tria,
     const std::string &           grid_generator_function_name,
     const std::string &           grid_generator_function_arguments);
+
+#ifdef DEAL_II_WITH_CGAL
+  /**
+   * Generate a Triangulation from the zero level set of an implicit function,
+   * using the CGAL library.
+   *
+   * This function is only implemented for `dim` equal to two or three, and
+   * requires that deal.II is configured using `DEAL_II_WITH_CGAL`. When `dim`
+   * is equal to three, the @p implicit_function is supposed to be negative in
+   * the interior of the domain, positive outside, and to be entirely enclosed
+   * in a ball of radius @p outer_ball_radius centered at the point
+   * @p interior_point. The triangulation that is generated covers the volume
+   * bounded by the zero level set of the implicit function  where the
+   * @p implicit_function is negative.
+   *
+   * When `dim` is equal to two, the generated surface triangulation is the zero
+   * level set of the @p implicit_function, oriented such that the surface
+   * triangulation has normals pointing towards the region where
+   * @p implicit_function is positive.
+   *
+   * The optional variable arguments @p cgal_args can be used to pass additional
+   * arguments to the CGAL::Mesh_criteria_3 class (see
+   * https://doc.cgal.org/latest/Mesh_3/index.html) for more information.
+   *
+   * The arguments allow for fine control on the size, quality, and distribution
+   * of the cells of the final triangulation. CGAL uses named parameters for
+   * these arguments in dimension three, i.e., they must be specified with the
+   * syntax `CGAL::parameters::parameter_name=parameter_value`, irrespective of
+   * their order. Accepted parameters are:
+   *
+   * - CGAL::parameters::edge_size: a scalar field (resp. a constant) providing
```

Comment:
I am not sure if you want to print this parameter (and all the ones in the following lists) in tt font,
```suggestion
   * - `CGAL::parameters::edge_size`: a scalar field (resp. a constant) providing
```
This would be consistent with how you write it in the documentation above.